monitoring/swarm.sh

```diff
@@ -56,6 +56,9 @@ function initialize_package() {
 
```

Comment:
There are more updates that should take place here, as we are making sure in this script that all the services are deployed correctly and also when destroy that all services, configs and volumes are removed:

- if there are services that are replicated mode (not global) we need to add their names in the top of the file (SCALED_SERVICES), and for cluster services inside the condition in line 26
- we need to add the other global services to the (SERVICE_NAMES) list
- We should add "promtail" and "loki" to the "prune_configs" list in line 95
- We should add these three volumes "loki_data" "minio1_data1" and "minio1_data2" to the list of removed volumes in line 89